net-mgmt/pfSense-pkg-snmptt/files/usr/local/pkg/snmptt.xml

```diff
@@ -0,0 +1,730 @@
+					Set to 0 to prevent unsafe REGEX code from being executed (default).<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>0</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Remove Backslash From Quotes</fielddescr>
+			<fieldname>remove_backslash_from_quotes</fieldname>
+			<description>
+				<![CDATA[
+					Set to 1 to have the backslash (escape) removed from quotes passed from<br/>
+					snmptrapd. For example, \" would be changed to just " Set to 0 to disable.<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>0</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Dynamic Nodes</fielddescr>
+			<fieldname>dynamic_nodes</fieldname>
+			<description>
+				<![CDATA[
+					Set to 1 to have NODES files loaded each time a trap is processed.<br/>
+					Set to 0 to have all NODES files loaded when the snmptt.conf files are loaded.<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>0</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Description Mode</fielddescr>
+			<fieldname>description_mode</fieldname>
+			<description>
+				<![CDATA[
+					This option allows you to use the $D substitution variable to include the<br/>
+					description text from the SNMPTT.CONF or MIB files.<br/>
+					Set to 0 to disable the $D substitution variable.  If $D is used, nothing<br/>
+					will be outputted.<br/>
+					Set to 1 to enable the $D substitution variable and have it use the<br/>
+					descriptions stored in the SNMPTT .conf files.  Enabling this option can<br/>
+					greatly increase the amount of memory used by SNMPTT.<br/>
+					Set to 2 to enable the $D substitution variable and have it use the<br/>
+					description from the MIB files.  This enables the UCD-SNMP / NET-SNMP Perl<br/>
+					module save_descriptions variable.  Enabling this option can greatly<br/>
+					increase the amount of memory used by the Net-SNMP SNMP Perl module, which<br/>
+					will result in an increase of memory usage by SNMPTT.<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>0</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+				<option><name>2</name><value>2</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Description Clean</fielddescr>
+			<fieldname>description_clean</fieldname>
+			<description>
+				<![CDATA[
+					Set to 1 to remove any white space at the start of each line from the MIB<br/>
```

Comment:
Put short descriptions of these behaviors in the "name" part of the options list.